quiz.js

```diff
@@ -49,11 +84,14 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update({
+    name: "House Arryn"
+  }, {
+    $unset: { motto: 1
+  }
+});
 
```

Comment:
Correct. Though, for clarity, it might have been better to pick an empty string as the value to pair with `motto` - that might make it more clear that you're deleting the property.